packages/components/src/components/modal/modal.jsx

```diff
@@ -159,19 +159,21 @@ Modal.Body = Body;
 Modal.Footer = Footer;
 
 Modal.defaultProps = {
-    has_close_icon: true,
+    has_close_icon            : true,
```

Comment:
@zaki-binary you don't need to define a new prop? you can use  `has_close_icon` cause usually when you have close icon means you can close on outside click too, and if you don't have close_icon then you shouldn't click on outside click, its redundant